### DISCUSSION TITLE
[nuxt] error caught during app initialization H3Error: Page not found: /sw.js

### DISCUSSION THREAD

**Discussion by stephencarico** [nuxt/nuxt#19574] [Questions]
This warning chunk keeps clogging up my terminal _**only when I'm on port:3000**_. My workaround is to give my Nuxt app a different port to use.

I tried restarting my Macbook, but with no luck. This is happening with a brand new Nuxt 3 application when I try to create a `pages/index.vue` with `<NuxtPage />` in `App.vue`

_Does anyone know why this is happening and perhaps how to resolve?_

❌ **Port 3000**

<img width="1680" alt="Screenshot 2023-03-09 at 8 49 35 PM" src="https://user-images.githubusercontent.com/32233894/224219755-057f68b0-8afd-4b75-bd81-382cfad330bb.png">

✅ **Port XXXX**

<img width="1680" alt="Screenshot 2023-03-09 at 8 50 10 PM" src="https://user-images.githubusercontent.com/32233894/224219758-98654589-3e29-42ce-8f14-033e71f4160f.png">

**Operating System:** Ventura 13.2.1
**Processor:** i7

**Error message:**

```
[nuxt] error caught during app initialization H3Error: Page not found: /sw.js                                                                                                 20:50:47
    at Module.createError (file:///Users/stephencarico/Development/_templates/delete-this/node_modules/h3/dist/index.mjs:128:15)
    at /Users/stephencarico/Development/_templates/delete-this/node_modules/nuxt/dist/pages/runtime/plugins/router.mjs:141:113
    at triggerAfterEach (file:///Users/stephencarico/Development/_templates/delete-this/node_modules/vue-router/dist/vue-router.mjs:3306:13)
    at file:///Users/stephencarico/Development/_templates/delete-this/node_modules/vue-router/dist/vue-router.mjs:3209:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /Users/stephencarico/Development/_templates/delete-this/node_modules/nuxt/dist/pages/runtime/plugins/router.mjs:157:7
    at async createNuxtAppServer (/Users/stephencarico/Development/_templates/delete-this/node_modules/nuxt/dist/app/entry.mjs:30:7)
    at async default (file:///Users/stephencarico/Development/_templates/delete-this/node_modules/@nuxt/vite-builder/dist/runtime/vite-node.mjs:27:18)
    at async Object.renderToString (file:///Users/stephencarico/Development/_templates/delete-this/node_modules/vue-bundle-renderer/dist/runtime.mjs:172:19)
    at async file:///Users/stephencarico/Development/_templates/delete-this/.nuxt/dev/index.mjs:717:21 {
  statusCode: 404,
  fatal: false,
  unhandled: false,
  statusMessage: 'Page not found: /sw.js',
  __nuxt_error: true
}
```

**Comment by danielroe** [comment]
This should be resolved in the [edge channel](https://nuxt.com/docs/guide/going-further/edge-channel/), or in the next release.

**Reply by stephencarico** [reply]
I opted into the edge channel and did a fresh install of all of my node_modules, but I am seeing the same result. 

<img width="1680" alt="Screenshot 2023-03-10 at 5 03 40 PM" src="https://user-images.githubusercontent.com/32233894/224451431-21a5c4f0-bdeb-4b72-9fd4-b323749fce15.png">

**Reply by stephencarico** [reply]
When reverting back to `"nuxt": "3.1.1"` the error disappears.

<img width="1680" alt="Screenshot 2023-03-10 at 5 11 12 PM" src="https://user-images.githubusercontent.com/32233894/224451981-6413085e-55a1-434f-ac6e-d0e45ce83233.png">

**Reply by nawroz-m** [reply]
I've got the same error, with a brand new nux3 project
"nuxt": "^3.2.3"

**Reply by mktcode** [reply]
Same here. Seems to be resolved in 3.3.2.

**Reply by Rohithgilla12** [reply]
I think it's reintroduced in `3.5.0`
